client/src/components/Home/Home.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useContext, useState, useEffect } from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+} from 'react-router-dom';
+import {
+  Tab, Paper, Tabs,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import Events from '../Events/Events.jsx';
+import Flights from '../Flights/Flights.jsx';
+import Hotels from '../Hotels/Hotels.jsx';
```

Comment:
These appear to be unused in this file

Reply:
Still weeding out what we are using in the future.